client/hosting-features/components/hosting-features.tsx

```diff
@@ -70,17 +69,17 @@ const HostingFeatures = () => {
 			supportContext: 'site-monitoring-metrics',
 		},
 		{
-			title: translate( 'PHP Logs' ),
-			text: translate( 'View and download PHP error logs to diagnose and resolve issues quickly.' ),
-			supportContext: 'site-monitoring-logs',
-		},
-		{
-			title: translate( 'Server Logs' ),
+			title: translate( 'Logs' ),
 			text: translate(
-				'Gain full visibility into server activity, helping you manage traffic and spot security issues early.'
+				'View and download PHP error and web server logs to diagnose and resolve issues quickly.'
```

Comment:
```suggestion
				'View and download PHP Logs and Webserver Logs to diagnose and resolve issues quickly.'
```

Reply:
Same here! The actual subtabs are called `PHP error` and `web server`. The ones in the current docs are outdated.